cloud-shell/cloudbuild.yaml

```diff
@@ -12,10 +12,36 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The cloudbuild instructions originally generated by GCP to rebuild one image
+# look like below, for reference:
+
+# steps:
+# - name: 'gcr.io/cloud-builders/docker'
+#   args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', '.']
+# - name: 'gcr.io/cloudshell-images/custom-image-validation'
+#   args: ['image_test.py', '--image', 'gcr.io/$PROJECT_ID/$REPO_NAME']
+# images: ['gcr.io/$PROJECT_ID/$REPO_NAME']
+# timeout: '1200s'
+
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', '.']
-- name: 'gcr.io/cloudshell-images/custom-image-validation'
-  args: ['image_test.py', '--image', 'gcr.io/$PROJECT_ID/$REPO_NAME']
-images: ['gcr.io/$PROJECT_ID/$REPO_NAME']
-timeout: '1200s'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |-
+    git clone https://github.com/GoogleCloudPlatform/cloud-ops-sandbox.git
+    cd cloud-ops-sandbox
```

Comment:
It occurred to me that we should probably rebuild latest here too, before iterating over the tags:

```
// build latest image
docker build -t gcr.io/$PROJECT_ID/cloudshell-image:latest ./cloud-shell
docker push gcr.io/$PROJECT_ID/cloudshell-image:latest
```

Reply:
Good point! I'll do that.